sdk/search/search-documents/src/geographyPoint.ts

```diff
@@ -33,7 +33,7 @@ export default class GeographyPoint {
   public toJSON(): Record<string, unknown> {
     return {
       type: "Point",
-      coordinates: [this.longitude, this.latitude],
+      coordinates: [this.latitude, this.longitude],
```

Comment:
this is not correct: https://tools.ietf.org/html/rfc7946#appendix-A.1

The array is [longitude, latitude]:

>   A position is an array of numbers.  There MUST be two or more
   elements.  The first two elements are longitude and latitude, or
   easting and northing, precisely in that order and using decimal
   numbers.  Altitude or elevation MAY be included as an optional third
   element.

Reply:
The problem is that we copied our constructor order from .NET: https://docs.microsoft.com/en-us/dotnet/api/microsoft.spatial.geographypoint.create?view=odata-spatial-7.0 which is latitude, longitude.

But in our serialization code we have:

```ts
if (isGeoJSONPoint(input)) {
    return new GeographyPoint(input.coordinates[0], input.coordinates[1]);
  }
```

when it should be

```ts
if (isGeoJSONPoint(input)) {
    return new GeographyPoint(input.coordinates[1], input.coordinates[0]);
  }
```

Since we're in beta though - could we perhaps change the constructor of `GeographyPoint` to take an object `{latitude: number, longitude: number}` so these are no longer passed in confusingly by order?

Since I think we can all agree that this looks better:

```ts
if (isGeoJSONPoint(input)) {
    return new GeographyPoint({longitude: input.coordinates[0], latitude: input.coordinates[1]});
  }
```

Reply:
Done